src/classes/Api.php

```diff
@@ -160,4 +167,29 @@ public function reset() {
             $this->adapter = NULL;
         }
 
+        /**
+         * @inheritdoc
+         */
+        public function getFormFields($recipientlistId)
+        {
+
+            $cacheFilePath = get_temp_dir() . 'rapidmail_fields.php';
```

Comment:
I think you'd need a unique filename here as get_temp_dir() might fallback to sys_get_temp_dir() which could mess up caches on shared systems?!

Reply:
Changed to JSON.